crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -118,4 +118,14 @@ impl TransactionSubmitting for CustomNodesApi {
             })),
         }
     }
+
+    fn submission_status(&self, settlement: &Settlement) -> SubmissionLoopStatus {
+        if !settlement.mev_safe() {
+            return SubmissionLoopStatus::Disabled(
+                DisabledReason::CustomNodesDisabledMevExtractable,
+            );
+        }
+
+        SubmissionLoopStatus::Enabled
+    }
```

Comment:
This would effectively disable `TransactionStrategyArg::CustomNodes` until the follow-up PR is merged.
Depending on how long that is expected to take, I think it would be better to simply return `SubmissionLoopStatus::Enabled` here and leave a `TODO` comment to replace the implementation later on.
In that case it would also make sense to drop `mev_safe()` from this PR and only introduce it in the follow-up PR.

Reply:
Indeed, `mev_safe` is not relevant, but let's keep it for the idea to be more clear, I don't plan to merge this PR before the following PR for `mev_safe` is done, anyways.
`mev_safe` changed to `true` to avoid disabling `CustomNodes`.